packages/e2e/package.json

```diff
@@ -110,4 +110,4 @@
     "webpack": "^5.72.0",
     "webpack-cli": "^4.9.2"
   }
-}
+}
```

Comment:
Please remove

Reply:
`e2e package.json` change is entirely omitted after rebasing with master, af689c041ef2ff4ef70ca8d5452c2416a7906baa